scenes/Player/Player.gd

```diff
@@ -13,7 +12,11 @@ onready var flame_exhaust: Node2D = $Ship_Exhaust
 func _ready() -> void:
 	flame_exhaust.hide()
 
+
 func _physics_process(delta: float) -> void:
+	if PlayerState.hull_health <= 0: 
+		queue_free()
```

Comment:
`queue_free` changes the behavior when you die by resetting the camera back to the starting position and making it so the camera no longer moves when you press W/A/S/D. I think this is an improvement over the way things worked before but always good to call out changes like this in the PR description.

Reply:
Good advice, thanks. I forgot to mention that change.